src/api/app/helpers/webui/notification_excerpt_helper.rb

```diff
@@ -23,7 +23,7 @@ def excerpt(notification)
   private
 
   def truncate_to_first_new_line(text)
-    first_new_line_index = text.index("\n")
+    first_new_line_index = text.to_s.index("\n") # sometimes text can be nil
```

Comment:
It will never be nil if you just do ` notification.notifiable.body.to_s` and ` notification.notifiable.reason.to_s` above...

Reply:
Yes, I know, I was DRYing

Reply:
This has nothing to do with the DRY principle but okay. I don't care too much...